src/Widget/ButtonLink.php

```diff
@@ -11,4 +11,43 @@ class ButtonLink extends ActionLink
         'class'            => 'button-link',
         'data-base-target' => '_main'
     ];
+
+    /** @var bool Whether the button is disabled */
+    protected bool $isDisabled = false;
+
+    /**
+     * Set the disabled state of the button
+     *
+     * @param bool $disabled default true
+     *
+     * @return $this
+     */
+    public function setDisabled(bool $disabled = true): self
+    {
+        $this->isDisabled = $disabled;
+
+        return $this;
+    }
+
+    /**
+     * Get whether the button is disabled
+     *
+     * @return bool
+     */
+    public function isDisabled(): bool
+    {
+        return $this->isDisabled;
+    }
```

Comment:
I don't like the idea of having a button (link) that is disabled but doesn't have any explanation for this. Since the linked Notifications Web pull request adds a `title` attribute on its own, if it disables the button, I'd suggest to introduce another method: `ButtonLink::disable(string $explanation): static`